packages/big-design/src/components/Worksheet/styled.tsx

```diff
@@ -0,0 +1,22 @@
+import { theme as defaultTheme, remCalc } from '@bigcommerce/big-design-theme';
```

Comment:
I think we should be trying to use `remCalc` from the `theme` object instead of importing it 🤔  Did you see any other usage like this?

Reply:
Honestly I don't remember using it from the `theme` object in BigDesign before, I might need to take a look at other components at a later time.